### ISSUE TITLE
Image zoom buttons do not have an accessible name - Shopify Product Slider

### ISSUE THREAD

**Issue by aspatino** [metafizzy/flickity#1275]
Hello,

We have discovered an accessibility issue on the Flickity slider utilized in the Impulse theme by Archetype on our Shopify store. This appears in the <button> element for image zoom on product images in the shop (e.g. https://www.soapcauldron.com/products/essential-oil-lavender) and is flagged by Google Lighthouse and Axe Dev Tools. This button does not have an accessible name; the zoom functionality is not announced by screen readers. I believe all that is required is to add an aria-label or title to the button element.

I was unable to find an existing issue, or a suitable codepen to include a reduced test case here—please let me know if I missed that, or if this issue is better addressed elsewhere. Thank you!
